client/src/lib/api/datasets.ts

```diff
@@ -22,16 +24,20 @@ export async function createDataset(
     formData.append('elanOptions', JSON.stringify(elanOptions));
   }
 
-  return fetch(server + urls.api.datasets, {
+  return fetch(url, {
     method: 'POST',
     mode: 'cors',
     body: formData,
   });
 }
 
 export async function deleteDataset(name: string): Promise<Response> {
-  return fetch(`${server}${urls.api.datasets}${name}`, {
+  return fetch(`${url}${name}`, {
     mode: 'cors',
     method: 'DELETE',
   });
 }
+
+export async function downloadDataset(name: string): Promise<Response> {
+  return fetch(`${url}download/${name}`);
```

Comment:
Do we want to change the location of the / to be more consistent with the rest of the urls and the in-between bits